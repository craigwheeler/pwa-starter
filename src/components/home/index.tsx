import * as React from 'react';
import styled from 'styled-components';

export interface PropType {
  message: string;
}

export const Home = (props: PropType) => {
  return (
    <Container>
      <h1>{props.message}</h1>
    </Container>
  );
};

const Container = styled.div`
  display: flex;
  justify-content: center;
  align-items: center;
  height: 100vh;
  background-color: #999;
`;
