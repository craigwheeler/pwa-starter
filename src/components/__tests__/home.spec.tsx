import * as React from 'react';
import { render } from '@testing-library/react';
import { Home } from '../home';

describe('Home', () => {
  it('should render home page', () => {
    const elem = render(<Home message={'World'} />).container;
    expect(elem).toMatchSnapshot();
  });
});
