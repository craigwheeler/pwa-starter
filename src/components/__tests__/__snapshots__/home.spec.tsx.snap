// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render home page 1`] = `
<div>
  <h1>
    World
  </h1>
</div>
`;
